### DISCUSSION TITLE
Bug reports tab

### DISCUSSION THREAD

**Discussion by Multi-Coder-24** [modrinth/.github#160] [Feature requests]
I would like to see a simple bug tracker similar to github's solution for projects that don't have a public repository. And because people would have used modrinth to download the mod this would give the user an easy place to send bug reports.

I think this would definitely be a welcome feature for many as some projects licence is closed source so there is no other real place for bug reports to go other than the mod creator's specific idea for bug tracking which would cause the end user much trouble to have to go to another place just to send a bug report. For projects with a public repository this isn't much of an issue however I can still see benefits of having the bug tracker in one place

**Comment by Onako2** [comment]
Bigger closed-source projects usually have a repository just for issues or a Discord server but it is not perfect
